product_secondary_unit/models/product_secondary_unit_mixin.py

```diff
@@ -76,6 +76,11 @@ def _compute_helper_target_field_qty(self):
         for rec in self:
             if not rec.secondary_uom_id:
                 continue
```

Comment:
Add this or similar to avoid cache missed error:
```suggestion
                rec[rec._secondary_unit_fields["qty_field"]] = rec._origin[rec._secondary_unit_fields["qty_field"]]
                continue
```